core/includes/file.inc

```diff
@@ -1881,6 +1881,9 @@ function file_validate_size(File $file, $file_limit = 0, $user_limit = 0) {
  */
 function file_validate_is_image(File $file) {
   $errors = array();
+  if ('image/svg+xml' === mime_content_type($file->uri)) {
```

Comment:
Instead of this, have you considered, adding the svg support to `image_get_info` so that svg support is more integrated? A good reason to make this more integrated is that the toolkits might also support various svg commands. I think ImageMagick might.

Reply:
Made sense, done.